java/code/src/com/redhat/rhn/taskomatic/TaskoXmlRpcHandler.java

```diff
@@ -213,13 +215,24 @@ private TaskoBunch doBasicCheck(Integer orgId, String bunchName, String jobLabel
     public Integer unscheduleBunch(Integer orgId, String jobLabel) {
         // one or none shall be returned
         List<TaskoSchedule> scheduleList = TaskoFactory.listActiveSchedulesByOrgAndLabel(orgId, jobLabel);
+        TriggerKey triggerKey;
         Trigger trigger;
         try {
-            trigger = SchedulerKernel.getScheduler().getTrigger(triggerKey(jobLabel,
-                    TaskoQuartzHelper.getGroupName(orgId)));
+            triggerKey = triggerKey(jobLabel, TaskoQuartzHelper.getGroupName(orgId));
+            trigger = SchedulerKernel.getScheduler().getTrigger(triggerKey);
+
+            // Try to find retry triggers as fallback
+            if (trigger == null) {
+                triggerKey = SchedulerKernel.getScheduler()
+                    .getTriggerKeys(GroupMatcher.anyGroup()).stream()
```

Comment:
If I got it right this will get all the existing triggers and iterate through them. Do we have an estimation of how big this can be for large-scale environments?

Reply:
ok, sounds good to me. Lets mode forward with it and if we found any performance problem reach accordingly :)